examples/oemof_0.2/basic_example/basic_example.py

```diff
@@ -0,0 +1,213 @@
+    actual_value=data['demand_el'], fixed=True, nominal_value=1)}))
+
+# create simple transformer object representing a gas power plant
+energysystem.add(solph.Transformer(
+    label="pp_gas",
+    inputs={bgas: solph.Flow()},
+    outputs={bel: solph.Flow(nominal_value=10e10, variable_costs=50)},
+    conversion_factors={bel: 0.58}))
+
+# create storage object representing a battery
+storage = solph.components.GenericStorage(
+    nominal_capacity=10077997,
+    label='storage',
+    inputs={bel: solph.Flow(variable_costs=10e10)},
+    outputs={bel: solph.Flow(variable_costs=10e10)},
+    capacity_loss=0.00, initial_capacity=0,
+    nominal_input_capacity_ratio=1/6,
+    nominal_output_capacity_ratio=1/6,
+    inflow_conversion_factor=1, outflow_conversion_factor=0.8,
+)
+
+energysystem.add(storage)
+
+##########################################################################
+# Optimise the energy system and plot the results
+##########################################################################
+
+logging.info('Optimise the energy system')
+
+# initialise the operational model
+om = solph.Model(energysystem)
+
+# This is for debugging only. It is not(!) necessary to solve the problem and
+# should be set to False to save time and disc space in normal use. For
+# debugging the timesteps should be set to 3, to increase the readability of the
+# lp-file.
+if debug:
+    filename = os.path.join(
+        helpers.extend_basic_path('lp_files'), 'storage_invest.lp')
+    logging.info('Store lp-file in {0}.'.format(filename))
+    om.write(filename, io_options={'symbolic_solver_labels': True})
+
+# if tee_switch is true solver messages will be displayed
+logging.info('Solve the optimization problem')
+om.solve(solver=solver, solve_kwargs={'tee': solver_verbose})
+
+logging.info('Store the energy system with the results.')
+
+# Add results to the energy system
+energysystem.results['main'] = processing.results(om)
+energysystem.results['meta'] = processing.meta_results(om)
+
+# Store energy system with results
+# The default path is the '.oemof' folder in your $HOME directory.
+# The default filename is 'es_dump.oemof'.
+# You can omit the attributes (as None is the default value) for testing cases.
+# You should use unique names/folders for valuable results to avoid overwriting.
+energysystem.dump(dpath=None, filename=None)
+
+# ****************************************************************************
+# ********** PART 2 - Processing the results *********************************
+# ****************************************************************************
+
+logging.info('**** The script can be divided into two parts here.')
+logging.info('Restore the energy system and the results.')
+energysystem = solph.EnergySystem()
+energysystem.restore(dpath=None, filename=None)
+
+# check if the new result object is working for custom components
+results = energysystem.results['main']
+meta_results = energysystem.results['meta']
+
```

Comment:
More explanation what is done might to understand what is done  here. Something like "looking into specific parts of the energy system for a certain day in the year and displaying the values"